integ-test/src/test/java/org/opensearch/sql/legacy/OpenSearchSQLRestTestCase.java

```diff
@@ -201,6 +199,22 @@ protected static void wipeAllOpenSearchIndices(RestClient client) throws IOExcep
     }
   }
 
+  protected static void configureClient(RestClientBuilder builder, Settings settings)
```

Comment:
can you javadoc what you're doing here and why?

Reply:
Updated in a618be8f6